shared/src/packages/trucks/libs/validation-schemas/truck-add.validation-schema.ts

```diff
@@ -0,0 +1,70 @@
+import joi from 'joi';
+
+import {
+  FormLabel,
+  TruckManufacturer,
+  TruckTowType,
+  TruckYear,
+} from '../enums/enums.js';
+import { type TruckAddRequestDto } from '../types/truck-add-request-dto.type.js';
+
+const truckAddValidationSchema = joi.object<TruckAddRequestDto, true>({
+  manufacturer: joi.object({
+    label: joi.string().required(),
+    value: joi.valid(...Object.values(TruckManufacturer)).messages({
+      'any.only': `Invalid ${FormLabel.MANUFACTURER}`,
+      'any.required': `${FormLabel.MANUFACTURER} is required`,
+    }),
+  }),
+
+  capacity: joi
+    .number()
+    .min(0)
+    .required()
+    .messages({
+      'number.base': `${FormLabel.CAPACITY} must be a number`,
+      'number.min': `${FormLabel.CAPACITY} must be at least 0`,
+    }),
+
+  pricePerKm: joi
+    .number()
+    .precision(1)
+    .min(1)
+    .max(100)
```

Comment:
I already wrote about it - min/max should be shared between FE and Shared module